data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/BulkRetryStrategy.java

```diff
@@ -184,20 +182,28 @@ private void incrementErrorCounters(final Exception e) {
     }
 
     public void execute(final AccumulatingBulkRequest bulkRequest) throws InterruptedException {
-        // Exponential backoff run forever
-        // TODO: replace with custom backoff policy setting including maximum interval between retries
-        final BackOffUtils backOffUtils = new BackOffUtils(
```

Comment:
Can we now remove `BackOffUtils`? Is it being used elsewhere?

Reply:
Removed BackOffUtils.